OpenAI_API/Chat/Conversation.cs

```diff
@@ -122,5 +122,52 @@ public async Task<string> GetResponseFromChatbot()
 			}
 			return null;
 		}
+
+		/// <summary>
+		/// Calls the API to get a response, which is appended to the current chat's <see cref="Messages"/> as an <see cref="ChatMessageRole.Assistant"/> <see cref="ChatMessage"/>, and streams the results to the <paramref name="resultHandler"/> as they come in. <br/>
+		/// If you are on the latest C# supporting async enumerables, you may prefer the cleaner syntax of <see cref="StreamResponseEnumerableFromChatbot"/> instead.
+		///  </summary>
+		/// <param name="resultHandler">An action to be called as each new result arrives.</param>
+		public async Task StreamResponseFromChatbot(Action<string> resultHandler)
+		{
+			await foreach (string res in StreamResponseEnumerableFromChatbot())
+			{
+				resultHandler(res);
+			}
+		}
+
+		/// <summary>
+		/// Calls the API to get a response, which is appended to the current chat's <see cref="Messages"/> as an <see cref="ChatMessageRole.Assistant"/> <see cref="ChatMessage"/>, and streams the results as they come in. <br/>
+		/// If you are not using C# 8 supporting async enumerables or if you are using the .NET Framework, you may need to use <see cref="StreamResponseFromChatbot"/> instead.
+		/// </summary>
+		/// <returns>An async enumerable with each of the results as they come in.  See <see href="https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#asynchronous-streams"/> for more details on how to consume an async enumerable.</returns>
+		public async IAsyncEnumerable<string> StreamResponseEnumerableFromChatbot()
```

Comment:
Please use suffix `Async`

Reply:
`Conversation`'s `async Task<string> GetResponseFromChatbot()` doesn't have `Async` suffix. But some async methods in other classes have. So I am a little bit confused.

Reply:
I added "Async" suffix